content/docs/400-guides/400-observability/400-telemetry/200-metrics.mdx

```diff
@@ -427,3 +427,28 @@ FrequencyState {
 }
 */
 ```
+
+## Tagging Metrics
+When creating metrics, you can add tags to them. Tags are key-value pairs that provide additional context, helping in categorizing and filtering metrics. This makes it easier to analyze and monitor specific aspects of your application.
+
+
+### Tagging multiple Metrics
+
+Use `Effect.tagMetrics` to apply tags to all metrics created in the same context. This is useful for adding common tags that apply to multiple metrics.
+
+```ts twoslash
```

Comment:
Adding `twoslash` here and below will cause errors since it requires valid code inside the code fence

Reply:
better now?

Reply:
@tobiaslins it was ok before latest change:

this is ok
```ts
import { Metric } from "effect"

const counter = Metric.counter("request_count").pipe(
  Metric.tagged("environment", "production")
)
```
but this doesn't type check
```ts
import { Metric, Effect } from "effect"

const taskCount = Metric.counter("task_count")

Effect.gen(function* () {
  yield* taskCount(Effect.sleep("100 millis"))
}).pipe(
  Effect.tagMetrics("environment", "production")
)
```
the previous version does type check though
```ts
import { Metric, Effect } from "effect"

const taskCount = Metric.counter("task_count")
const task1 = Effect.succeed(1).pipe(Effect.delay("100 millis"))

Effect.gen(function* () {
  yield* taskCount(task1)
}).pipe(Effect.tagMetrics("environment", "production"))
```

Reply:
aaargh sorry. should finally be it now